cpu/ebizzy.py

```diff
@@ -0,0 +1,77 @@
+import os
+import json
+import re
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Ebizzy(Test):
+
+    '''
+    ebizzy is designed to generate a workload resembling common web application
+    server workloads. It is highly threaded, has a large in-memory working set,
+    and allocates and deallocates memory frequently.
+    '''
+
+    def setUp(self):
+        '''
+        Build ebizzy
+        Source:
+        http://liquidtelecom.dl.sourceforge.net/project/ebizzy/ebizzy/0.3
+        /ebizzy-0.3.tar.gz
+        '''
+        sm = SoftwareManager()
+        if not sm.check_installed("gcc") and not sm.install("gcc"):
+            self.error("Gcc is needed for the test to be run")
+
+        tarball = self.fetch_asset('http://liquidtelecom.dl.sourceforge.net'
+                                   '/project/ebizzy/ebizzy/0.3'
+                                   '/ebizzy-0.3.tar.gz')
+        data_dir = os.path.abspath(self.datadir)
+        archive.extract(tarball, self.srcdir)
+        cb_version = os.path.basename(tarball.split('.tar.')[0])
```

Comment:
`cb` is the `compilebench` abbreviation. It has no meaning for this test. Do you mind in rename it?